models/falcon-7b-instruct.json

```diff
@@ -0,0 +1,86 @@
+{
+  "_descriptorVersion": "0.0.1",  
+  "datePublished": "2023-10-31T16:01:50",
+  "name": "falcon-7b-instruct",
+  "description": "Falcon-7B-Instruct, based on Falcon-7B, has been fine-tuned on chat and instruct datasets. It offers outstanding performance and is an excellent choice for chat and instruct applications. Falcon-7B-Instruct is part of the Falcon family of language models, known for their exceptional capabilities and openness.",
```

Comment:
I typically generate description by following this process, more or less:
- copy the parts from the original model card with information about the model, the dataset, training process, prompting nuances, license
- feed this information to GPT4 along with a prompt that looks something like:


> Please generate a short summary of this information. The audience is AI model users with varying technical backgrounds, from non-technical users to ML researchers. The summary must include every important detail in the text above. Use a scientific and dispassionate tone.


feel free to tweak it